core/templates/third-party-imports/ui-codemirror.import.ts

```diff
@@ -17,9 +17,15 @@
  */
 
 window.CodeMirror = require('codemirror');
+window.diff_match_patch = require('diff-match-patch');
+window.DIFF_DELETE = require('diff-match-patch');
+window.DIFF_EQUAL = require('diff-match-patch');
+window.DIFF_INSERT = require('diff-match-patch');
 
 import 'codemirror/mode/javascript/javascript';
 import 'codemirror/mode/python/python';
 import 'codemirror/mode/yaml/yaml';
-require('static/diff-match-patch-1.0.0/diff_match_patch.js');
+import 'diff-match-patch';
 import 'codemirror/addon/merge/merge';
+import 'codemirror/addon/merge/merge.css';
+import 'codemirror/lib/codemirror.css';
```

Comment:
[Optional] Maybe you can create a new import file, that imports code mirror and imports diff-match-patch library in the file.

Reply:
undoing the diff-match-patch changes here